pkg/tsdb/influxdb/influxql/converter/converter.go

```diff
@@ -345,11 +345,8 @@ func typeOf(value interface{}) data.FieldType {
 
 func handleTimeSeriesFormatWithTimeColumn(valueFields data.Fields, tags map[string]string, columns []string, measurement string, frameName []byte, query *models.Query) []*data.Frame {
 	frames := make([]*data.Frame, 0, len(columns)-1)
-	for i, v := range columns {
-		if v == "time" {
-			continue
-		}
-		formattedFrameName := string(util.FormatFrameName(measurement, v, tags, *query, frameName[:]))
+	for i := 1; i < len(columns); i++ {
```

Comment:
Your commit message was helpful; it might be a good code comment.

```suggestion
	// don't iterate over first column as it is a time column already
	for i := 1; i < len(columns); i++ {
```